lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -197,6 +216,14 @@ function distributeTwoNodes(firstNode, secondNode) {
   return Node.Creator.parenthesis(Node.Creator.operator('+', newArgs));
 }
 
+function hasFraction(args) {
+  return args.filter(isFraction).length > 0;
+}
+
+function isFraction(node) {
```

Comment:
I thought this might be a util function somewhere but I couldn't find it. Do you think maybe we should export functions like this in some kind of util so they can be used around the codebase?